### PR TITLE
Updating the application

### PR DESCRIPTION
**Pull request by pranavbaburaj**
A command is added to check if the user has the latest version of the app
`dynamic -u` or `dynamic --update`

But this feature requires you to create a new release, else this feature won't work. But, it will not create errors. It just logs that `the application does not have any releases`

### CODE REVIEW

**Comment by GouravSardana** [comment]
@pranavbaburaj you have one trailing space and you have to put one extra line. 
See the details in the codacy logs

**Comment by pranavbaburaj** [comment]
> @pranavbaburaj you have one trailing space and you have to put one extra line.
> See the details in the codacy logs

OK :slight_smile:

**Review by anshsrtv — APPROVED** [review]
Great job @pranavbaburaj! This will be useful in the long run!

**Review by honeybhardwaj — APPROVED** [review]
LGTM !!

**Review by Siddhant-K-code — APPROVED** [review]
LGTM! Good Work 🎉

File: src/arguments/update.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        via github releases API
+        """
+        self.current_version = current_version
+        self.release_api_url = "https://api.github.com/repos/IndianOpenSourceFoundation/dynamic-cli/releases/latest"
+
+    def check_for_updates(self):
+        try:
+            data = requests.get(self.release_api_url)
+            data = data.json()
+
```
Comment (Siddhant-K-code) [inline src/arguments/update.py:20]: it would be good if you can remove this Trailing whitespace

File: src/arguments/update.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import requests as requests
+from termcolor import colored
+import webbrowser as webbrowser
+
+from .error import SearchError
+
+class UpdateApplication(object):
+    def __init__(self, current_version):
+        """
```
Comment (Siddhant-K-code) [inline src/arguments/update.py:9]: Add a blank line b/w summary line and description

Comment (GouravSardana) [reply src/arguments/update.py:9]: +1

Comment (pranavbaburaj) [reply src/arguments/update.py:9]: Ohh, fine. Thank you

**Review by Siddhant-K-code — APPROVED** [review]
LGTM! Good Work

**Review by GouravSardana — APPROVED** [review]

**Pull request merged**